Uplift/Controllers/HomeViewController+Extensions.swift

```diff
@@ -38,11 +38,17 @@ extension HomeViewController: UICollectionViewDataSource {
             cell.delegate = self
             cell.configure(for: gymClassInstances)
             return cell
+        case .lookingFor:
+            // swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Constants.lookingForListCellIdentifier, for: indexPath) as! LookingForListCell
+            cell.delegate = self
+            cell.configure(for: lookingForCategories, width: collectionView.bounds.width)
+            return cell
```

Comment:
Instead of creating a cell and returning it within each switch case, why not define the cell outside the switch, set it to a value and modify it inside the switch, and then return the cell outside the switch?

Reply:
The dequeued cells are all of different types, so it makes more sense to create a cell object declared as a cell type rather than create a generic cell object and repeatedly downcast it.